### PR TITLE
Update HTML Layout and add new HTML templates

### PR DESCRIPTION
**Pull request by asantos96**
Update HTML Layout and add new HTML templates for driver to client maps

### CODE REVIEW

File: src/main/resources/static/app.css [hunk 1]
```diff
@@ -3,7 +3,7 @@
     text-align: center;
     color: white;
 
-    background-color: blue;
+    background-color: black;
```
Comment (ruqayyahmustafa2022) [inline src/main/resources/static/app.css:6]: great choice of color!

Comment (asantos96) [reply src/main/resources/static/app.css:6]: Thanks! 😊

**Review by ruqayyahmustafa2022 — CHANGES_REQUESTED** [review]
Can you change black to orange?

**Comment by asantos96** [comment]
> Can you change black to orange?

Anyway we can keep the black? (Test Convo)

**Comment by asantos96** [comment]
Based on verbal conversation on 4/14 we decided to stick with black.

**Pull request merged**